### PR TITLE
feat(reverse-proxy): document how to set Bonita behing firewall

### PR DESCRIPTION
**Pull request by educhastenier**
into DMZ, behind a reverse proxy.

Closes [RUNTIME-157](https://bonitasoft.atlassian.net/browse/RUNTIME-157)

### CODE REVIEW

File: modules/ROOT/pages/reverse-proxy-configuration.adoc [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+Learn how to set up a reverse proxy to expose Bonita web interface publicly while keeping the backend safe.
+
+== Context
+
+[WARNING]
+====
+Bonitasoft does not support anymore having Bonita portal running on a dedicated Tomcat server on one side, and Bonita Engine running on a dedicated Tomcat server on the other side.
+====
+
+If you have to expose Bonita Portal or a Bonita-developed application publicly, but legibly want to protect any other resources behind your firewall, you should set up a software architecture similar to this one:
```
Comment (abirembaut) [inline modules/ROOT/pages/reverse-proxy-configuration.adoc]: legibly or legitimately ?

File: modules/ROOT/pages/reverse-proxy-configuration.adoc [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+    <Location />
+        Order allow,deny
+        Allow from all
+    </Location>
+
+</VirtualHost>
+----
+
+Be sure to also xref:ssl.adoc#_tomcat_and_ssl_offloading[configure Tomcat IP Valve] for IP forwarding.
```
Comment (abirembaut) [inline modules/ROOT/pages/reverse-proxy-configuration.adoc]:
```suggestion
Make sure to also xref:ssl.adoc#_tomcat_and_ssl_offloading[configure Tomcat IP Valve] for IP and protocol forwarding.
```

**Review by abirembaut — COMMENTED** [review]
LGTM
just 2 comments

File: modules/ROOT/pages/reverse-proxy-configuration.adoc (hunk 1 above)
Comment (educhastenier) [reply modules/ROOT/pages/reverse-proxy-configuration.adoc]: Nice catch, I didn't know that in English

Comment (educhastenier) [inline modules/ROOT/pages/reverse-proxy-configuration.adoc]:
```suggestion
If you have to expose Bonita Portal or a Bonita-developed application publicly, but legitimately want to protect any other resources behind your firewall, you should set up a software architecture similar to this one:
```

Comment (educhastenier) [reply modules/ROOT/pages/reverse-proxy-configuration.adoc]: changed

**Pull request merged**